critiquebrainz/db/users.py

```diff
@@ -0,0 +1,35 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(item):
+    url = "https://gravatar.com/avatar/{0}{1}"
+    email = item['email']
+    if email and item['show_gravatar']:
+        return url.format(md5(email.encode('utf-8')).hexdigest(), "?")
+    else:
+        return url
+
+
+def get_many_by_mb_username(usernames):
+    """Get information on users
+    Args:
+        usernames (list): MusicBrainz usernames.
+    Returns:
+        All columns of "user" table (list of dictionaries)
+        and avatar_url (Gravatar url)
+
+    Ignores missing users
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+          SELECT "id", "display_name", "email", "created",
+                  "musicbrainz_id", "show_gravatar"
+          FROM "user"
```

Comment:
What about `SELECT *`? You may implement new values in `user` model, and it would be much easier to access in future.